web-app/src/main/webapp/index.html

```diff
@@ -1,89 +1,84 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="stylesheet" href="css/skeleton.css">
-    <link rel="stylesheet" href="css/normalize.css">
-    <script src="script.js"></script>
-    <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDtnOIOa1q7Mwn8OeRkeXdNPxpbaWT2ZFw&callback=initMap" 
-      type="text/javascript"></script>
-  </head>
-  <body>
-    <!-- Search bar -->
-    <div class="container">
-      <form id="address-form">
-        <div class="row">
-          <div class="two-thirds columns">
-            <input class="u-full-width" type="text" placeholder="Search"
-              id="search-location">
-          </div>
+<head>
+  <meta charset="UTF-8" />
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="css/style.css" />
+  <link rel="stylesheet" href="css/skeleton.css" />
+  <link rel="stylesheet" href="css/normalize.css" />
+  <script src="script.js"></script>
+  <script async defer
+    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDtnOIOa1q7Mwn8OeRkeXdNPxpbaWT2ZFw&callback=initMap"
+    type="text/javascript"></script>
+</head>
+<body>
+  <!-- Search bar -->
+  <div class="container">
+    <form id="address-form">
+      <div class="row">
+        <div class="two-thirds columns">
+          <input class="u-full-width" type="text" placeholder="Search" id="search-location" />
```

Comment:
Just FYI: there is no such thing as a closing tag in html5 (and the <!DOCTYPE html> declaration at the top of this file means this is written in html5). As far as I know, html5 parsers will simply ignore the `/`. It's fine if you think it is more readable this way.

Reply:
Thanks for letting me know! I think I might leave them in there for readability, if that's not considered bad practice.